src/tasks/AutoHedge.py

```
@@ -47,6 +47,21 @@ def __init__(self, *args, **kwargs):
         self.roulette_task = None
         self.maze_task = None
 
+    def _parse_serum_pct(self, text: str) -> int | None:
+        # OCR 结果可能包含前后缀与置信度等信息，例如："血清35%_1.00"
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)